Components/Grids/grid.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="The Grid component of Ozone-UI">
+    <meta name="author" content="Govardhan Reddy">
+    <title>Ozone UI | Grids</title>
+    <link rel="stylesheet" href="../../Docs/doc.css">
+    <link rel="stylesheet" href="grid.css">
+</head>
+<body>
+    <nav class='nav'>
+        <div >
```

Comment:
Here you can use Header tag instead of div for semantic checkpoints.

Reply:
okay Aditi